src/nohang

```diff
@@ -3958,29 +3966,6 @@ if check_kmsg:
 if ZFS:
     log('WARNING: ZFS found. Available memory will not be calculated '
         'correctly (issue#89)')
-    try:
-        # find indexes
-        with open(arcstats_path, 'rb') as f:
-            a_list = f.read().decode().split('\n')
-        for n, line in enumerate(a_list):
-            if line.startswith('c_min '):
-                c_min_index = n
-
-            elif line.startswith('size '):
-                size_index = n
-
-            elif line.startswith('arc_meta_used '):
-                arc_meta_used_index = n
-
-            elif line.startswith('arc_meta_min '):
-                arc_meta_min_index = n
-
-            else:
-                continue
-    except Exception as e:
-        log(e)
-        ZFS = False
```

Comment:
instead of storing these into individual variables and writing fairly complicated code to parse this file again, we have a method that stores the whole contents of the file into an easy-to-read dict.